lte/gateway/c/oai/tasks/grpc_service/AmfServiceImpl.cpp

```diff
@@ -1,18 +1,30 @@
 /*
-Copyright 2020 The Magma Authors.
-This source code is licensed under the BSD-style license found in the
-LICENSE file in the root directory of this source tree.
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 
 #include <string>
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+#include "amf_service_handler.h"
+#include "log.h"
+#ifdef __cplusplus
+}
+#endif
+
 #include "AmfServiceImpl.h"
 #include "lte/protos/session_manager.pb.h"
```

Comment:
what is being used from session_manager.pb.h?

Reply:
will be taken out once amf.proto is defined